exercises/practice/parallel-letter-frequency/parallel_letter_frequency.rb

```diff
@@ -0,0 +1,6 @@
+=begin
+Write your code for the 'Parallel Letter Frequency' exercise in this file. Make the tests in
+`parallel_letter_frequency_test.rb` pass.
+To get started with TDD, see the `README.md` file in your
+`ruby/parallel_letter_frequency` directory.
+=end
```

Comment:
```suggestion
=begin
Write your code for the 'Parallel Letter Frequency' exercise in this file. Make
the tests in `parallel_letter_frequency_test.rb` pass.

To get started with TDD, see the `README.md` file in your
`ruby/parallel_letter_frequency` directory.
=end

```

EOL and 80 column for source code.